Makefile

```diff
@@ -82,4 +82,10 @@ docs: .docker-build
 	${DOCKERCOMPOSE} run web $(MAKE) -C docs/ clean
 	${DOCKERCOMPOSE} run web $(MAKE) -C docs/ html
 
+createsuperuser: .docker-build
+	@echo "Create a superuser account for Standup local dev."
+	${DOCKERCOMPOSE} run web ./manage.py createsuperuser
```

Comment:
If they do this first it's possible they'll not have run `manage.py migrate` yet right? Perhaps we should have a `make migrate` that depends on `.docker-build` and have this depend on `migrate`?